internal/service/batch/job_definition.go

```diff
@@ -409,6 +517,21 @@ func findJobDefinition(ctx context.Context, conn *batch.Batch, input *batch.Desc
 	return output.JobDefinitions[0], nil
 }
 
+func ListActiveJobDefinitionByName(ctx context.Context, conn *batch.Batch, name string) ([]*batch.JobDefinition, error) {
```

Comment:
Is there a particular reason we define this as a public function? Do we use it in other packages?

Reply:
Yes, its used in the test function `testAccCheckJobDefinitionDestroy`